aerospike/check.py

```diff
@@ -3,75 +3,173 @@
 
 # stdlib
 import socket
+import time
+import re
 from contextlib import closing
+from collections import namedtuple
 
 # project
 from checks import AgentCheck
 
-EVENT_TYPE = SOURCE_TYPE_NAME = 'aerospike'
-SERVICE_CHECK_NAME = '%s.server_up' % EVENT_TYPE
+SOURCE_TYPE_NAME = 'aerospike'
+SERVICE_CHECK_NAME = '%s.cluster_up' % SOURCE_TYPE_NAME
+CLUSTER_EVENT_TYPE = SOURCE_TYPE_NAME
```

Comment:
Are these really event types? Or are they more metric types?

Reply:
yes. should be to event type.